generator/nidaqmx_python_generator/metadata/attributes.py

```diff
@@ -0,0 +1,130 @@
+attributes = {
+    'Channel': {
+         6145: {
+            'access': 'read-write',
+            'name': 'AI_ATTEN',
+            'resettable': True,
+            'type': 'float64',
+            'ctypesDataType': 'ctypes.c_double',
```

Comment:
Should scrapigen keys be in camelCase or snake_case? I'm not sure if there is a precedent.

Reply:
Was there a precedent? (i.e. did the existing scrapigen output have any camelCase or snake_case names?)

Reply:
as mentioned, I noticed some other [existing keys related to grpc proto generation](https://github.com/ni/grpc-device-scrapigen/blob/30746b2ee2a166ee3dfcd8f8a77550a5e85c8a9f/out/daq/export/functions.py#L5677) being snake case, therefore I switched my implementation to snake case as well.